src/adapter/adapterDiscovery.ts

```diff
@@ -0,0 +1,439 @@
+            vendorId: '10c4',
+            productId: 'ea60',
+            // manufacturer: '',
+            // /dev/serial/by-id/usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_a215650c853bec119a079e957a0af111-if00-port0
+            pathRegex: '.*slzb-07.*',
+        },
+        {
+            // Sonoff ZBDongle-E V2
+            vendorId: '1a86',
+            productId: '55d4',
+            manufacturer: 'ITEAD',
+            // /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20240122184111-if00
+            // /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_186ff44314e2ed11b891eb5162c61111-if00-port0
+            pathRegex: '.*sonoff.*plus.*',
+        },
+        // {
+        //     // TODO: Z-station by z-wave.me (EFR32MG21A020F1024IM32)
+        //     vendorId: '',
+        //     productId: '',
+        //     // manufacturer: '',
+        //     // /dev/serial/by-id/usb-Silicon_Labs_CP2105_Dual_USB_to_UART_Bridge_Controller_012BA111-if01-port0
+        //     pathRegex: '.*CP2105.*',
+        // },
+    ],
+    zstack: [
+        {
+            // ZZH
+            vendorId: '0403',
+            productId: '6015',
+            manufacturer: 'Electrolama',
+            pathRegex: '.*electrolame.*', // TODO
+        },
+        {
+            // slae.sh cc2652rb
+            vendorId: '10c4',
+            productId: 'ea60',
+            // manufacturer: '',
+            pathRegex: '.*2652.*',
+        },
+        {
+            // Sonoff ZBDongle-P (CC2652P)
+            vendorId: '10c4',
+            productId: 'ea60',
+            // manufacturer: '',
+            // /dev/serial/by-id/usb-Silicon_Labs_Sonoff_Zigbee_3.0_USB_Dongle_Plus_0111-if00-port0
+            // /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_b8b49abd27a6ed11a280eba32981d111-if00-port0
+            pathRegex: '.*sonoff.*plus.*',
+        },
+        {
+            // CC2538
+            vendorId: '0451',
+            productId: '16c8',
+            manufacturer: 'Texas Instruments',
+            // zStack30x: /dev/serial/by-id/usb-Texas_Instruments_CC2538_USB_CDC-if00
+            pathRegex: '.*CC2538.*',
+        },
+        {
+            // CC2531
+            vendorId: '0451',
+            productId: '16a8',
+            manufacturer: 'Texas Instruments',
+            // /dev/serial/by-id/usb-Texas_Instruments_TI_CC2531_USB_CDC___0X00124B0018ED1111-if00
+            pathRegex: '.*CC2531.*',
+        },
+        {
+            // CC1352P_2
+            vendorId: '0451',
+            productId: 'bef3',
+            manufacturer: 'Texas Instruments',
+            pathRegex: '.*CC1352P_2.*', // TODO
+        },
+        {
+            // CC26X2R1
+            vendorId: '0451',
+            productId: 'bef3',
+            manufacturer: 'Texas Instruments',
+            pathRegex: '.*CC26X2R1.*', // TODO
+        },
+        {
+            // SMLight slzb-07p7
+            vendorId: '',
```

Comment:
@darkxst Can you help in completing some of these fingerprints if you have some time?

Reply:
This is standard cp2102 usb id: `10c4:ea60`
Manufacturer: `SMLIGHT`
Product: `SMLIGHT_SLZB-07p7`

Later when p10 is available it will just change product string
Product: `SMLIGHT_SLZB-07p10`

Reply:
Also Mg24 is out now.

product string: `SMLIGHT_07Mg24` all other details same as SLZB-07